droidlet/memory/sql_memory.py

```diff
@@ -997,8 +1001,50 @@ def db_write(self, query: str, *args) -> int:
         if self.on_delete_callback is not None and deleted:
             self.on_delete_callback(deleted)
         self._db_write("DELETE FROM Updates")
+        # JSONify formatted data to send to dashboard timeline
+        query_table, query_operation = self.parse_sql(query[:query.find("(") - 1])
+        query_dict = self.format_query(query, *args)
+        hook_data = {
+            "name" : "db_write", 
+            "time" : self.get_time(), 
+            "table_name" : query_table, 
+            "operation" : query_operation, 
+            "args" : query_dict, 
+            "result" : r
+        }
+        hook_data = json.dumps(hook_data, default=str)
+        self.dispatch_signal.send(self.db_write, data=hook_data)
         return r
 
+    def parse_sql(self, query):
```

Comment:
`parse_sql` and `format_query` should be moved to a separate file (`droidlet/memory/memory_util.py`). @aszlam  wdyt?